running-pixelfed/installation.md

```diff
@@ -99,17 +99,15 @@ If you are running Redis on another machine:
 
 If you are using a Unix socket for Redis, then:
 
-- Set `REDIS_HOST` to the path of the socket, e.g. `/run/redis/redis.sock`
-- Set `REDIS_PORT` to `null`
-
-If you are using a Unix socket 
+- Set `REDIS_SCHEME` to `unix`
+- Set `REDIS_PATH` to the path of the socket, e.g. `/run/redis/redis.sock`
 
 ::: tip TCP server vs. Unix socket
 Redis usually comes pre-configured to listen for TCP requests on the local machine over port 6379. In your Redis configuration, typically at `/etc/redis.conf`, the relevant lines are `bind 127.0.0.1` and `port 6379`.
 
 Changing the latter line to `port 0` will disable TCP listening, in which case Redis must be configured for socket access. Lines such as `unixsocket /run/redis/redis.sock` and `unixsocketperm 770` must be set to enable socket access. Additionally, both the app user and web user should have permission to access the socket, e.g. by being added to the `redis` group.
 
-Using a Unix socket is optional, but may provide faster access since it does not have to create TCP packets. TCP is usually used over a network, and would be required if Redis were running on a different machine than your web server.
+Using a Unix socket is optional, but may provide faster access since it does not have to create TCP packets. TCP is usually used over a network, and would be required if Redis were running on a different machine than your web server. Unix sockets are also helpful for running more than one Redis instance on the same machine, which you might want to do if you have more than one application using Redis (a single Redis instance [should not be used](https://redis.io/commands/select) by multiple unrelated applications).
```

Comment:
Couldn't one also theoretically run multiple instances of Redis on the same machine via multiple TCP ports?

Reply:
Absolutely, but if you're not going to use the default Redis listening configuration anyway, you might as well make it listen on a Unix socket. (That _is_ why I just said "it's helpful" though instead of saying that was _the_ way to do it. I can rephrase if you want.)